tests/integration/golang/mlflow/artifact/list_s3_test.go

```diff
@@ -41,9 +42,20 @@ func (s *ListArtifactS3TestSuite) SetupTest() {
 	s3Client, err := helpers.NewS3Client(helpers.GetS3EndpointUri())
 	assert.Nil(s.T(), err)
 
+	createBuckets(s3Client)
```

Comment:
maybe we can put all of that into `BaseTestSuite` so we can:
- create `s3` client in centralised place.
- setup `buckets` in centralised place.
- we can use `TearDownSuite` to clean something we need.

Reply:
i moved to tests/integration/golang/helpers/s3.go